application/routes/api.php

```diff
@@ -14,5 +15,10 @@
 |
 */
 
-Route::get('/v1/classifier-values', [ClassifierValueController::class, 'index']);
-Route::get('/v1/classifier-values/{id}', [ClassifierValueController::class, 'get']);
+Route::get('/classifier-values', [ClassifierValueController::class, 'index']);
+Route::get('/classifier-values/{id}', [ClassifierValueController::class, 'show'])->whereUuid('id');
+
+Route::withoutMiddleware(['auth:api', 'throttle:api'])->group(function () {
+    Route::get('/sync/classifier-values', [ClassifierValueSyncController::class, 'index']);
+    Route::get('/sync/classifier-values/{id}', [ClassifierValueSyncController::class, 'show'])->whereUuid('id');
+});
```

Comment:
I think it would be good to have auth middleware applied globally and endpoints that should be publicly accessible will be using the withoutmiddleware. The reason for it is that the whole system in general requires authentication to access it and there's currently no functionalities that don't require authentication except the login page. This would also decrease the risk that some endpoint will be left unguarded